generic/tests/cfg/iometer_windows.cfg

```diff
@@ -73,3 +73,37 @@
                     bg_mode = yes
                 - after_test:
                     bg_mode = no
+        - with_multi_scsi_disks:
+            only virtio_scsi
+            run_timeout = 3600
+            icf_name = "iometer_multi_disks.icf"
+            format_multi_disks = yes
+            disk_indexes = "1 2 3 4 5 6"
+            disk_letters = "I J K L M N"
+            disk_fstypes = "fat32 fat ntfs fat32 fat ntfs"
```

Comment:
also here why repeat each file system types? and missed type Res~

Reply:
ok, will it by the manual case.